File: csharp/Example2.cs
using System;
using Rinearn.ExevalatorCS;

/// <summary>
/// An example to use various operators and parentheses.
/// </summary>
static class Example2
{
    public static void Main()
    {
        // Create an instance of Exevalator Engine
        Exevalator exevalator = new Exevalator();

        // Evaluate the value of an expression
        double result = exevalator.Eval("(-(1.2 + 3.4) * 5) / 2");

        // Display the result
        Console.WriteLine("Result: " + result);
    }
}

File: java/Example2.java
// import anypackage.in.which.you.put.Exevalator;

/**
 * An example to use various operators and parentheses.
 */
public class Example2 {

    public static void main(String[] args) {

        // Create an instance of Exevalator Engine
        Exevalator exevalator = new Exevalator();

        // Evaluate the value of an expression
        double result = exevalator.eval("(-(1.2 + 3.4) * 5) / 2");

        // Display the result
        System.out.println("result: " + result);
    }
}
